app/src/main/java/org/hackillinois/android/view/schedule/ScheduleFragment.kt

```diff
@@ -8,12 +8,15 @@ import androidx.fragment.app.FragmentPagerAdapter
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Button
 import kotlinx.android.synthetic.main.fragment_schedule.view.*
 import org.hackillinois.android.R
 import org.hackillinois.android.viewmodel.ScheduleViewModel
 
 class ScheduleFragment : Fragment() {
 
+    private lateinit var favoriteScheduleButton: Button
```

Comment:
I would just call this variable `favoriteButton`. You are in the `ScheduleFragment` class, so the original variable name is a little redundant.